rllib/tuned_examples/bc/benchmark_atari_pong_bc.py

```diff
@@ -0,0 +1,257 @@
+    .evaluation(
+        evaluation_interval=3,
+        evaluation_num_env_runners=1,
+        evaluation_duration=5,
+        evaluation_parallel_to_training=True,
+    )
+    .learners(
+        num_learners=args.num_gpus if args.num_gpus > 1 else 0,
+        num_gpus_per_learner=1,
+    )
+    # Note, the `input_` argument is the major argument for the
+    # new offline API. Via the `input_read_method_kwargs` the
+    # arguments for the `ray.data.Dataset` read method can be
+    # configured. The read method needs at least as many blocks
+    # as remote learners.
+    .offline_data(
+        input_=destination_file_name,
+        input_read_method="read_tfrecords",
+        input_read_method_kwargs={
+            # Note, `TFRecords` datasets in `rl_unplugged` are GZIP
+            # compressed and Arrow needs to decompress them.
+            "arrow_open_stream_args": {"compression": "gzip"},
+            # Use enough reading blocks to scale well.
+            "override_num_blocks": 20,
+            # TFX improves performance extensively. `tfx-bsl` needs to be
+            # installed for this.
+            "tfx_read_options": TFXReadOptions(
+                batch_size=2000,
+            ),
+        },
+        # `rl_unplugged`'s data schema is different from the one used
+        # internally in `RLlib`. Define the schema here so it can be used
+        # when transforming column data to episodes.
+        input_read_schema={
+            Columns.EPS_ID: "episode_id",
+            Columns.OBS: "o_t",
+            Columns.ACTIONS: "a_t",
+            Columns.REWARDS: "r_t",
+            Columns.NEXT_OBS: "o_tp1",
+            Columns.TERMINATEDS: "d_t",
+        },
+        # Increase the parallelism in transforming batches, such that while
+        # training, new batches are transformed while others are used in updating.
+        map_batches_kwargs={"concurrency": max(args.num_gpus * 20, 20)},
+        # When iterating over batches in the dataset, prefetch at least 20
+        # batches per learner. Increase this for scaling out more.
+        iter_batches_kwargs={
+            "prefetch_batches": max(args.num_gpus * 20, 20),
+            "local_shuffle_buffer_size": None,
+        },
+    )
+    .training(
+        # To increase learning speed with multiple learners,
+        # increase the learning rate correspondingly.
+        lr=0.0008 * max(1, args.num_gpus**0.5),
+        train_batch_size_per_learner=2000,
+        # Use the defined learner connector above, to decode observations.
+        learner_connector=_make_learner_connector,
+    )
+    .rl_module(
+        model_config_dict={
+            "vf_share_layers": True,
+            "conv_filters": [[16, 4, 2], [32, 4, 2], [64, 4, 2], [128, 4, 2]],
+            "conv_activation": "relu",
+            "post_fcnet_hiddens": [256],
+            "uses_new_env_runners": True,
+        }
+    )
+)
+
+# TODO (simon): Change to use the `run_rllib_example` function as soon as tuned.
```

Comment:
nit: TODO? Or still WIP?

Reply:
WIP :) It is still not running smoothly and for debugging I am not using tune.